release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -93,6 +93,13 @@ You can now use the {product-title} Custom Metrics Autoscaler Operator to dynami
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-autoscaling-ingress-controller[Autoscaling an Ingress Controller].
 
+[id="ocp-4-12-nw-ingress-haproxy-maxconn-default"]
+==== HAProxy maxConnections default is now 50,000
+
+In {product-title} 4.12, the default value for the `maxConnections` setting is now 50,000. Previously starting with {product-title} 3.5, the default value for the maxconn setting was 20,000.
```

Comment:
@skrthomas, is it `50,000` or `50000`? There's probably no comma in the actual value.

> Previously starting with {product-title} 3.5, the default value for the maxconn setting was 20,000.

Is it `maxconn` or `maxConnections`? Or was it the former in OCP 3.x and the latter in OCP 4.x?

Thanks!

Reply:
Its without the comma good catch! So, upon further thought re the 3.x vs 4.x discussion, I think I'm going to omit that whole bit about 3.x and just say previously it was `20000` because in 3.5 is was `ROUTER_MAX_CONNECTIONS`, then it dropped off and wasn't introduced in 4.x until 4.11. So yeah, I don't even think its worth the confusion of mentioning 3.x here.

Reply:
@jboxman does `maxConnections` need to be in backticks both times?

Reply:
@skrthomas, if this is a specific configuration field, I'd probably use backticks or write it as _max connections_ perhaps with the assumption that someone knows that means `maxConnections` mentioned elsewhere in this entry.